website/static/js/myProjects.js

```diff
@@ -1836,7 +1813,8 @@ var Information = {
                         m('[role="tabpanel"].tab-pane.active#tab-information',[
                             m('p.db-info-meta.text-muted', [
                                 m('', 'Visibility : ' + (item.attributes.public ? 'Public' : 'Private')),
-                                m('', 'Category: ' + categoryMap(item.attributes.category)),
+                                m('.capitalize', 'Category: ' + item.attributes.category),
+                                m('.capitalize', 'My permission: ' + permission),
```

Comment:
Where is the `.capitalize` rule defined in our CSS? I'm having trouble finding it. It looks like it may be coming from a library we're using, like maybe Bootstrap, but in Bootstrap the class is [`.text-capitalize`](http://getbootstrap.com/css/#type-transformation).